payments-core/src/main/java/com/stripe/android/model/parsers/PaymentMethodWithLinkDetailsJsonParser.kt

```diff
@@ -11,24 +11,31 @@ internal object PaymentMethodWithLinkDetailsJsonParser : ModelJsonParser<Payment
 
     override fun parse(json: JSONObject): PaymentMethod? {
         val paymentMethod = PaymentMethodJsonParser().parse(json.getJSONObject("payment_method"))
+        val linkPaymentDetailsJson = json.optJSONObject("link_payment_details")
 
-        val consumerPaymentDetails = if (FeatureFlags.linkPMsInSPM.isEnabled) {
```

Comment:
The feature flag usage is no longer necessary, since `PaymentMethodWithLinkDetailsJsonParser` is only ever called if the feature flag is enabled.